apps/src/blockly/addons/cdoSerializationHelpers.ts

```diff
@@ -166,7 +185,7 @@ function adjustBlockPositions(blocks, workspace) {
 
     // Set initial position; collision area must be updated to account for new position
     // every time block is moved
-    block.moveTo({x, y});
+    block.moveTo(new Blockly.utils.Coordinate(x, y));
```

Comment:
`moveTo` [expects a Coordinate](https://developers.google.com/blockly/reference/js/blockly.blocksvg_class.moveto_1_method.md). Our previous method worked because the [method](https://github.com/google/blockly/blob/a517317b77c1e58727b748a99f9ae70b2c27d26a/core/block_svg.ts#L440) only references `.x` and `.y`, but it isn't guaranteed to continue to work so we should use the correct object